### PR TITLE
feat(ng) enable strict templates

### LINKED ISSUE #1181: fullTemplateTypeCheck and strictTemplates
When using LF with fullTemplateTypeCheck and strictTemplates, we have a bunch of errors when using
- `<lu-user-tile />`
- ` | luUserDisplay`

Because they expect a `ILuUser`. The thing is that they work with only a `firstName` and a `lastName`. We could use `Partial<ILuUser>` or a new type.

### PR DESCRIPTION
**Pull request by lucienbertin**
closes #1181

### CODE REVIEW

**Comment by LuccaIntegration** [comment]
jenkins auto deploy http://lucca-front.lucca.local/PR-1213

File: packages/ng/applications/sandbox/src/app/issues/date-picker-width/date-picker-width.component.ts [hunk 1]
```diff
@@ -5,6 +5,7 @@ import { Component } from '@angular/core';
 	templateUrl: './date-picker-width.component.html'
 })
 export class DatePickerWidthComponent {
+	true = true;
```
Comment (GuillaumeNury) [inline packages/ng/applications/sandbox/src/app/issues/date-picker-width/date-picker-width.component.ts:8]: wut ?

File: packages/ng/applications/sandbox/src/app/issues/select-overlap/select-overlap.component.ts [hunk 2]
```diff
@@ -8,4 +8,5 @@ export class SelectOverlapComponent {
 	model = 1;
 	item;
 	user;
+	true = true;
```
Comment (GuillaumeNury) [inline packages/ng/applications/sandbox/src/app/issues/select-overlap/select-overlap.component.ts:11]: wut ?

**Review by GuillaumeNury — COMMENTED** [review]
nice

**Comment by GuillaumeNury** [comment]
It is not solving #1181 issue.

File: packages/ng/applications/sandbox/src/app/issues/date-picker-width/date-picker-width.component.ts (hunk 1 above)
Comment (lucienbertin) [reply packages/ng/applications/sandbox/src/app/issues/date-picker-width/date-picker-width.component.ts:8]: not really needed, its in the sandbox so idc and dont feel like i need to fix it

**Comment by LuccaIntegration** [comment]
jenkins auto deploy http://lucca-front.lucca.local/PR-1213

**Pull request merged**